src/theory/uf/theory_uf.h

```diff
@@ -142,12 +155,6 @@ class TheoryUF : public Theory {
    */
   Node explain(TNode literal, eq::EqProof* pf);
 
-  /** Literals to propagate */
-  context::CDList<Node> d_literalsToPropagate;
-
-  /** Index of the next literal to propagate */
-  context::CDO<unsigned> d_literalsToPropagateIndex;
-
```

Comment:
Why did these get removed?

Reply:
They weren't used. If you don't want me to refactor/clean in the future, I can avoid this.
Personally I like to reduce code as I add, but I can see how this confuses pull requests.

Reply:
It's better to do clean-up as a separate PR.  I know it's a pain, but otherwise it does get confusing.